Add section for configuring event pruning in helm installation of trento (#198)
SUSE/doc-unversioned: trento/adoc/helm_event_pruning.adoc

pruning process that removes expired events.

The following configurations are available:

`+pruneEventsOlderThan+`::
  The number of days registered events are stored in the database. The
  default value is *10*. _Beware that `+pruneEventsOlderThan+` can be
  set to *0* but that would mean that all events are deleted every time
  the cron job runs, making it impossible to analyze/troubleshoot any
  issue with the application_
`+pruneEventsCronjobSchedule+`::
  The frequency of the cron job that deletes expired events. The default
  value is *"0 0 * * *"*, which runs daily at midnight.

To modify these default values, execute the following Helm command:

[source,bash]
----
helm ... \
    --set trento-web.pruneEventsOlderThan=<<EXPIRATION_IN_DAYS>> \
    --set trento-web.pruneEventsCronjobSchedule="<<NEW_SCHEDULE>>"
  
----

Replace the placeholders with the desired values:

`+EXPIRATION_IN_DAYS+`::
  Number of days to retain events in the database before pruning.
`+NEW_SCHEDULE+`::
  The cron expression specifying how frequently the pruning job should
  run.

*Example* command to retain events for 30 days and schedule pruning
daily at 3 AM:

[source,bash]

> ----
> helm install trento-server . \
>   --set trento-web.pruneEventsOlderThan=30 \
>   --set trento-web.pruneEventsCronjobSchedule="0 3 * * *"

Reviewer: ```suggestion
----
helm upgrade \
  --install trento-server oci://registry.suse.com/trento/trento-server \
  --set trento-web.trentoWebOrigin=yourtrentoserverhostname \
  --set trento-web.adminUser.password=youradminpasswor \
  --set trento-web.pruneEventsOlderThan=30 \
  --set trento-web.pruneEventsCronjobSchedule="0 3 * * *"
```

[[helm-event-pruning]]
== Configuring Event Pruning

The event pruning feature allows administrators to manage how long
registered events are stored in the database and the frequency of the
pruning process that removes expired events.

The following configurations are available:

`+pruneEventsOlderThan+`::
  The number of days registered events are stored in the database. The
  default value is *10*. _Beware that `+pruneEventsOlderThan+` can be
  set to *0* but that would mean that all events are deleted every time
  the cron job runs, making it impossible to analyze/troubleshoot any
  issue with the application_
`+pruneEventsCronjobSchedule+`::
  The frequency of the cron job that deletes expired events. The default
  value is *"0 0 * * *"*, which runs daily at midnight.

> To modify these default values, execute the following Helm command:
> 
> [source,bash]
> ----
> helm install trento-server . \
>     --set trento-web.pruneEventsOlderThan=<<EXPIRATION_IN_DAYS>> \
>     --set trento-web.pruneEventsCronjobSchedule="<<NEW_SCHEDULE>>" \
>     --set trento-web.adminUser.password=<<ADMIN_PASSWORD>>

Reviewer: ```suggestion
To modify these default values, add the following variables to the Helm installation command:

[source,bash]
----
helm ... \
    --set trento-web.pruneEventsOlderThan=<<EXPIRATION_IN_DAYS>> \
    --set trento-web.pruneEventsCronjobSchedule=<<NEW_SCHEDULE>>
```

  
----

Replace the placeholders with the desired values:

`+EXPIRATION_IN_DAYS+`::
  Number of days to retain events in the database before pruning.
`+NEW_SCHEDULE+`::
  The cron expression specifying how frequently the pruning job should
  run.

> `+ADMIN_PASSWORD+`::
>   The password for the admin user.

Reviewer: ```suggestion
```

*Example* command to retain events for 30 days and schedule pruning
daily at 3 AM:

[source,bash]
----

> helm install trento-server . \
>   --set trento-web.pruneEventsOlderThan=30 \
>   --set trento-web.pruneEventsCronjobSchedule="0 3 * * *" \
>   --set trento-web.adminUser.password=admin123

Reviewer: ```suggestion
helm ... \
  --set trento-web.pruneEventsOlderThan=30 \
  --set trento-web.pruneEventsCronjobSchedule="0 3 * * *"
```
